lib/modules/manager/gomod/artifacts.ts

```diff
@@ -271,7 +273,16 @@ export async function updateArtifacts({
 
     const execCommands: string[] = [];
 
-    let args = 'get -d -t ./...';
+    const userGoGetDirs = config.goGetDirs?.filter(isValidLocalPath);
```

Comment:
please make this fully conditional inside a single `if`, so we don't need so many conditional statements